Automattic/sensei
# Update font sizes for module title and lesson header

Resolves https://github.com/Automattic/themes/issues/7085

## Proposed Changes
* Updates font size of lesson header and Module title in Course theme

We've used the preset font sizes of course theme. We've used the **medium** font size for the module name and **small** font size for the lesson title.

For default variation of Course theme, we didn't change the font size as the medium of default was a bit smaller.

## Testing Instructions


1. Checkout this branch of Course theme https://github.com/Automattic/themes/pull/7465
2. Create a Course, in Course Outline block, add a few modules and lessons
3. Make sure the Lesson title looks bigger
4. Make sure the Module name is bigger than Lesson title
5. Check for all variations of the Course theme
6. Check with other themes to make sure nothing has broken with them.

## Screenshots:
![Current state](https://github.com/Automattic/sensei/assets/6820724/d98baaba-18d4-4b86-ae38-0f880660b738)
![Screenshot 2023-11-07 at 8 08 07 AM](https://github.com/Automattic/sensei/assets/6820724/b89097bf-e1ad-41cf-8bc8-bc61bca022d9)
![Screenshot 2023-11-07 at 8 08 29 AM](https://github.com/Automattic/sensei/assets/6820724/3486c400-b64f-49d6-bb10-62cd3d2631f3)
![Screenshot 2023-11-07 at 8 08 51 AM](https://github.com/Automattic/sensei/assets/6820724/5bad041e-c456-457f-8e58-cc6391d6ebc6)

## Pre-Merge Checklist
